setup.py

```diff
@@ -45,7 +45,7 @@ def write_version_py():
     package_data={'force_wfmanager.left_side_pane': 'icons/*'},
     entry_points={
         'gui_scripts': [
-            ('force_wfmanager = force_wfmanager.run:main')
+            ('force_wfmanager = force_wfmanager.cli.run:main')
```

Comment:
Ok, this addresses a question that @jjenthought raised about this specific issue. Good call. I didn't know that. We should certainly have gui then, instead of cli. Sorry @jjenthought

Reply:
Thanks for the advice! I think I got too caught up with the idea of making things 'user-friendly' with the compgen stuff. The idea was that it would show if the program was broken (test no.2) or if they had shell problems (test no.1) or hadn't loaded the edm environment. 
However I forgot: users won't actually be running these tests so no.1 is pretty redundant! Stefano has also suggested some additional ways to test things, since no.2 requires user input it's not so well suited for automated testing!
Since this change needs to be updated to incorporate some new tests, I'll switch the name over as well